integration_tests/src/main/python/regexp_test.py

```diff
@@ -956,3 +956,11 @@ def test_regexp_memory_ok():
             'spark.rapids.sql.batchSizeBytes': '20' # 1 row in the batch
         }
     )
+
+def test_re_replace_all():
+    # regression test for https://github.com/NVIDIA/spark-rapids/issues/8323
```

Comment:
Nitpick: Do we have a prescribed style for function documentation? I've been using the following even for tests:
```suggestion
    """
    regression test for https://github.com/NVIDIA/spark-rapids/issues/8323
    """
```

Reply:
Good point. Probably best to stick with the Python documentation approach.